cmd/application.go

```diff
@@ -74,6 +75,11 @@ var applicationDeleteCmd = &cobra.Command{
 		client := getOcClient()
 		appName := args[0]
 		var confirmDeletion string
+		// Project
+		currentProject := project.GetCurrent(client)
+		// Print App Information which will be deleted
+		err := printDeleteAppInfo(client, appName, currentProject)
```

Comment:
Shouldn't this reuse functions from #287?

Reply:
Text content is different, so I created separate function for this